### DISCUSSION TITLE
Confused

### DISCUSSION THREAD

**Discussion by Methen** [LegacyUpdate/LegacyUpdate#192] [Questions & Help]
Why is all the updates that iam getting  embedded and all of that and iam not even runng that os I running standard xp pro tha does not even make any sense

**Comment by kirb** [comment]
Sorry @Methen, I just realised I missed this post. If you're still wondering, this is because you selected the option in the Legacy Update setup to receive Windows Embedded 2009 updates, or something else you've installed already did that. These are extra updates that almost all work on regular XP Pro/Home, released after XP support was discontinued in 2014, continuing until Embedded 2009 support was discontinued in 2019.
